daemon/model-dbus-impl.cc

```diff
@@ -0,0 +1,269 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * @file test-dbus-impl.c
```

Comment:
Please change the file name.
```suggestion
 * @file model-dbus-impl.cc
```

Reply:
Nice catch! 👍